### PR TITLE
[OSDEV-673] Facility History. API endpoint times out

### PR DESCRIPTION
**Pull request by Innavin369**
[OSDEV-673](https://opensupplyhub.atlassian.net/browse/OSDEV-673) Facility History. API endpoint times out.

* fixed issue "TypeError: the JSON object must be str, bytes or bytearray, not list"

### CODE REVIEW

File: doc/RELEASE-NOTES.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * *Describe architecture/environment changes here.*
 
 ### Bugfix
-* *Describe bugfix here.*
+* [OSDEV-673](https://opensupplyhub.atlassian.net/browse/OSDEV-673) found and fixed “TypeError: the JSON object must be str, bytes or bytearray, not list“
```
Comment (vladsha-dev) [inline doc/RELEASE-NOTES.md:26]: Could you please briefly describe what is the problem you solved?  What has changed when you fixed this bug?
For instance, before the fix a user couldn't download the list, but after applying the fix they can.

**Review by vladsha-dev — CHANGES_REQUESTED** [review]
One minor comment.
Overall nice job!

File: src/django/api/tests/test_facility_history_endpoint.py [hunk 2]
```diff
@@ -1136,3 +1137,13 @@ def test_in_group_receives_200(self):
             history_response.status_code,
             200,
         )
+
+
+class DescribeChangeValueTests(FacilityAPITestCaseBase):
```
Comment (vladsha-dev) [inline src/django/api/tests/test_facility_history_endpoint.py:1142]: Each class should have a separate file or you can make these tests as part of the  FacilityHistoryEndpointTest class.

**Review by vladsha-dev — CHANGES_REQUESTED** [review]

File: doc/RELEASE-NOTES.md (hunk 1 above)
Comment (Innavin369) [reply doc/RELEASE-NOTES.md:26]: added description

File: src/django/api/tests/test_facility_history_endpoint.py (hunk 2 above)
Comment (Innavin369) [reply src/django/api/tests/test_facility_history_endpoint.py:1142]: Removed separate class, added tests to Facility history endpoint test

**Review by vladsha-dev — APPROVED** [review]
LGTM.
Nice job!

**Review by roman-stolar — APPROVED** [review]
LGTM

**Pull request merged**